install/kubernetes/helm/istio/charts/pilot/templates/meshexpansion.yaml

```diff
@@ -1,20 +1,20 @@
 {{- if .Values.global.meshExpansion }}
-
+{{ $istioNamespace := default .Chart.Namespace .Values.global.istioNamespace }}
```

Comment:
there is no definition in `values.yaml` of `global.istioNamespace`.  Hence this seems redundant.

Reply:
@sdake, thanks for the review.
I see it used in gateways chart:
```
./install/kubernetes/helm/istio/charts/gateways/templates/deployment.yaml:        {{- if $.Values.global.istioNamespace }}
./install/kubernetes/helm/istio/charts/gateways/templates/deployment.yaml:          - zipkin.{{ $.Values.global.istioNamespace }}:9411
./install/kubernetes/helm/istio/charts/gateways/templates/deployment.yaml:          {{- if $.Values.global.istioNamespace }}
./install/kubernetes/helm/istio/charts/gateways/templates/deployment.yaml:          - istio-pilot.{{ $.Values.global.istioNamespace }}:15011
./install/kubernetes/helm/istio/charts/gateways/templates/deployment.yaml:          {{- if $.Values.global.istioNamespace }}
./install/kubernetes/helm/istio/charts/gateways/templates/deployment.yaml:          - istio-pilot.{{ $.Values.global.istioNamespace }}:15010
```
And `global` is a specialcase in Helm as I recall. But sure thing I can remove it.

Reply:
hmm, that variable is not present in `values.yaml` - maybe a bug in the gateways chart.  I'm not sure how this works :)

Reply:
As I recall (I might be wrong) helm is gathering Values.global from all charts main and subs, combines them and then it's available for all charts. So you don't have to declare it in values.yaml, you can just use it :) 
Another thing, you don't have to declare a value in values.yaml to use it. Once you run helm with --set or -f, these specified values will be added into chart .Values and available to use.